docs/markdown/foundations/content_standards.md

```diff
@@ -0,0 +1,928 @@
+
+<TwoCol>
+  <Group>
+    <Do title="Do examples"/>
+     - Delete group board
+     - Enjoy!
+     - Keep scrolling to see more
+  </Group>
+  <Group>
+  <Dont title="Don't examples"/>
+     - How about deleting this group board?
+     - Are we having fun yet?
+     - Want more Pinterest?
+  </Group>
+</TwoCol>
+
+---
+
+### Settings
+
+<TwoCol>
+  <Group>
+    <Do title="Do"/>
+     - Use Yes | No for toggles
+     - Use sentence case for settings fields
+     - Use “you/your”
+     - Examples: 
+       - Also get an email when someone: Comments on your Pin
+  </Group>
+  <Group>
+  <Dont title="Don't"/>
+     - Don’t use On | Off for toggles
+     - Don’t use “me/mine”
+     - Examples:
+        - Also get an email when someone: Comments on my Pin
+  </Group>
+</TwoCol>
+
+---
+
+### Specific verbs
+
+In UI writing, sometimes it’s hard to choose between two seemingly similar verbs. Here are some tips:
+
+**Pick vs. choose vs. select:**
+Pick and choose sound more conversational in en_US/UK, and should always be used when discussing taste (“Pick your favorite topics!”) or other more human concepts. If you’re thinking about saying “choose,” try “pick” instead (unless you’re trying to avoid monotony). 
+
+*Pick:*
+- Pick 5 topics you love
+
+<br/>
+
+**See vs. view:**
+View derives from some action on the user's part (choosing filters, say) and See comes from work Pinterest’s already done for the user (like providing additional search results/related Pins).
+
+*See:*
+- Come see! (tooltip on more ideas tabs)
+- See ideas trending in Hounds (email notif)
+
+*View:*
+- View your map (map button CTA)
+- View in chronological order (made up filter sort)
+
+---
+
+### Tenses
+
+Use the [present perfect](https://www.grammarly.com/blog/present-perfect-continuous-tense/) tense in UI writing, even when using a contraction seems more conversational. 
+
+<TwoCol>
+  <Group>
+    <Do title="Do"/>
+     - You have a new follower
+  </Group>
+  <Group>
+  <Dont title="Don't"/>
+     - You've got a new follower
+  </Group>
+</TwoCol>
+
+---
+
+### Pin puns
+
+We all love a good Pin pun, but unfortunately, they don’t translate very well. We save Pin puns for internal comms, but don’t use them in external language or content. 
+
+---
+
+## More resources
+
+**Pinterest resources**
+- [Pinterest Content Design Hub](https://coda.io/d/Content-Design-Hub_d3AuBYHVHhZ/Content-Design-Hub_suVcA#_lucnd): A central hub for everything Content Design at Pinterest. 
+- [Pinterest Dictionary](https://docs.google.com/document/d/1JQWpTbnIj3w1lJtZoyYcOieSveOsGjuSeHqMdAfetU4/edit): A glossary of Content Design terminology at Pinterest. 
+- [Pinterest Brand Guidelines](https://brand.pinterest.com/): Standards for brand design at Pinterest. *Always check with the brand design team to make sure you're using the latest guidelines.*
```

Comment:
The first two internal docs need to be hidden behind Pinch links, and should also include the internal icon

Reply:
Done